pkg/tasks/alertmanager.go

```diff
@@ -178,6 +178,17 @@ func (t *AlertmanagerTask) create(ctx context.Context) error {
 			return errors.Wrap(err, "initializing Alertmanager object failed")
 		}
 
+		// Use console URL as Alertmanager ExternalURL since UI access to AlertManager is disabled.
+		consoleRoute, err := t.client.GetRouteWithNameAndNameSpace(ctx, "console", "openshift-console")
+		if err != nil {
+			return errors.Wrap(err, "getting console route failed")
+		}
+		consoleUrl, err := t.client.GetRouteURL(ctx, consoleRoute)
+		if err != nil {
+			return errors.Wrap(err, "getting console route URL failed")
+		}
+		a.Spec.ExternalURL = consoleUrl.String()
```

Comment:
Iiuc the backlink should contain a `monitoring` path component, like here https://github.com/openshift/cluster-monitoring-operator/pull/1532/files#diff-b0ca3a050a076f5a4007692cff8088723c9fe6f8c6734f8a8bc1bdd54f8d624cR105

Reply:
that's better, it redirects to the monitoring page. 👍